### PR TITLE
Add UsePascalCase setting for Enum members

### PR DESCRIPTION
**Pull request by makhin**
For now not possible to Generate enum members as is in the Database, but it is useful setting because not always enum should be translated to Pascal case. 
Use **Setting.PascalCase** for this behavior is also not good, because it should be independent settings.

### CODE REVIEW

**Comment by sjh37** [comment]
Thank you Alexandr. Well done. You are the first one to make use of BuiltTT to generate the TT file for you after using the Generator project. :-) Most just modify the `.ttinclude` file and leave it to me to add the code into the generator project.
I will give you a free licence and email it to amakhin@gmail.com as a thank you!

**Comment by makhin** [comment]
Thanks for the license I am very grateful to you, always try to understand the product developer's original idea and follow his code standards. I think about one more feature to add to your perfect product, mb I will create one more PR later.

File: Tester.Integration.EfCore3/File based templates/Templates.EF6/Enums.mustache [hunk 1]
```diff
@@ -1,6 +1,13 @@
-﻿public enum {{EnumName}}{{#newline}}
-{{{#newline}}
+﻿{{#each EnumAttributes}}
+{{this}}{{#newline}}
+{{/each}}
+public enum {{EnumName}}{{#newline}}
+{{{
```
Comment (sjh37) [inline Tester.Integration.EfCore3/File based templates/Templates.EF6/Enums.mustache:5]: bug

File: Tester.Integration.EfCore3/File based templates/Templates.EFCore3/Enums.mustache [hunk 2]
```diff
@@ -1,6 +1,13 @@
-﻿public enum {{EnumName}}{{#newline}}
-{{{#newline}}
+﻿{{#each EnumAttributes}}
+{{this}}{{#newline}}
+{{/each}}
+public enum {{EnumName}}{{#newline}}
+{{{
```
Comment (sjh37) [inline Tester.Integration.EfCore3/File based templates/Templates.EFCore3/Enums.mustache:5]: bug

**Comment by sjh37** [comment]
Now released in v3.5.0

**Pull request merged**